Contributing.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to the SPDX 3.0 Model
+
+## General
+SPDX is always welcoming new contributors! The discussions are happening on the spdx-tech mailing list
+and during our weekly meetings. All the details are in: https://spdx.dev/participate/tech/
+
+## Contributing to a specific profile
+There are multiple profiles being developed in parallel for the SPDX 3.0 Model. During this initial phase of development, each profile working group will contribute changes to its own branch in this repository. For example, any changes to the Build Profile should be committed to the `build-profile` branch. There will be one maintainer per profile in charge of merging any profile development changes to the profile-specific branch. Once the first "complete" version of a profile is ready, the profile maintainer will alert the general SPDX tech group that the profile model is ready for review.
```

Comment:
Suggest changing `should be committed to the` to `should be submitted as a change request to the`